src/Mod/CAM/Path/Base/Generator/helix.py

```diff
@@ -148,6 +163,26 @@ def helix_cut_r(r):
         arc_cmd = "G2" if direction == "CW" else "G3"
         commandlist.append(Path.Command("G0", {"X": startPoint.x + r, "Y": startPoint.y}))
         commandlist.append(Path.Command("G1", {"Z": startPoint.z}))
+
+        # FeedRateCheckbox.checked
+        if not feedRateAdj:
+            feedRateRatio = 0
+        else:
+            if (startAt == "Inside") or (
+                r == radii[0]
+            ):  # first cut, always adj for outer wall chip-load
+                feedRateRatio = r / (
+                    r + tool_radius
+                )  # classic (hole_rad-tool_rad)/hole_rad adjustment
+            else:
+                if tool_radius > 200 * r:
+                    feedRateRatio = 200  # prevent div zero
+                else:
+                    feedRateRatio = (
+                        r + tool_radius
+                    ) / r  # startAt outside: increase spindle feed rate to maintain inner-cut chip-load
```

Comment:
This would also benefit of using `elif`:

```suggestion
        if not feedRateAdj:
            feedRateRatio = 0
        elif (startAt == "Inside") or (r == radii[0]):
            # first cut, always adj for outer wall chip-load
            # classic (hole_rad-tool_rad)/hole_rad adjustment
            feedRateRatio = r / (r + tool_radius)
        elif tool_radius > 200 * r:
            feedRateRatio = 200  # prevent div zero
        else:
            # startAt outside: increase spindle feed rate to maintain inner-cut chip-load
            feedRateRatio = (r + tool_radius) / r
```

Reply:
@J-Dunn - participation in Open Source development requires a spirit of cooperation and good will. We ask that *any time* you are working with a reviewer on a PR their comments be taken with that spirit in mind. Deeply-nested code represents a significant cognitive load and immediate "code smell", and @hyarion's comment is not unreasonable. Please bear the [Code of Conduct](https://github.com/FreeCAD/FreeCAD/blob/main/CODE_OF_CONDUCT.md) in mind as you craft your responses to reviewer comments and keep them respectful, considerate, and forgiving.

Reply:
Anyone experiencing a significant cognitive load  when seeing deeply-nested code should probably take a break from auditing code. 

Personally I find indented blocks immediately  clear in structure in the bat of an eye. Non-indented elifs are nice if you have line length problems but they do require reading and mental parsing of the written code. 

That may be the way my mental processes work but that is why indenting was invented before python decided to make  an obligatory part the syntax rules: it is clearer and easier to read and interpret than non-indented code.